src/PKSim.Core/Commands/EditIndividualMoleculeExpressionInSimulationSubjectFromQueryCommand.cs

```diff
@@ -58,5 +63,17 @@ public override void Execute(IExecutionContext context)
          _queryExpressionResults = null;
          _simulationSubject = null;
       }
+
+      private void updateLocalizationForProtein(IndividualProtein protein, IExecutionContext context)
+      {
+         //special case for blood cells and vascular endothelium. if the value is set > 0. we need to turn on some locations programatically
+         var bloodCells = _queryExpressionResults.ExpressionResultFor(CoreConstants.Compartment.BLOOD_CELLS);
+         if (bloodCells?.RelativeExpression > 0 && !protein.IsBloodCell())
```

Comment:
why do we need the part `&& !protein.IsBloodCell()` ?  
(same question for VascEndo below)

Reply:
The logic that I have in mind is the following: If Bloodcells was already activated (Membrane OR Intracellular is on), then the field was set by the user and the rel exp is visible. Therefore, I do not want to trigger a command that willl reset potentially some fractions etc... 

In short, I just want to ensure that the rel exp field is made visible only if it was not already visible.

Reply:
The logic seems ok :)
But I still don'T understand the highlighted part. It checks if the protein is named "Blood Cells". Why?
I guess you mean ` !protein.InBloodCell()` ??

And also for VascEndo `!proteinInVascularEndothelium` and not `!proteinIsVascularEndothelium`?

Reply:
you are very much correct. I changed the code locally without saving and it did not commit those changes. I am going to add test for this use case where it's already set :)

Reply:
Very good eye @Yuri05 Thanks for catching this. 
I added a new test testing the second part of the if for VascularEndo and RBC . We are good